lib/components/input-group.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div :class="classObject">
+        <slot name="left"></slot>
```

Comment:
Making api complex for just special use cases is not also a good idea. just consider how much simpler is current usage than using bootstrap html markups directly. For multiply addons, we can support array values and rendering internally using `v-for`.